src/scripts.js

```diff
@@ -52,14 +54,31 @@ window.addEventListener('load', () => {
   }).catch(error => console.log(error))
 });
 
+
+
 //----DATA FUNCTIONS----
 
 const initialSetup = (travelers, destinations, trips) => {
+  dateFunction()
   createTraveler(travelers)
   createDestinations(destinations)
   createTrips(trips)
 };
 
+const dateFunction = () => {
+    let d = new Date()
+    let year = d.getFullYear().toString()
+    let month = d.getMonth().toString()
+    let day = d.getDate().toString()
+    if (month.length < 2) {
+    month = '0' + month;
+    }
+    if (day.lenght < 2) {
+      day = '0' + day;
+    }
+    date = [year, month, day].join('/')
```

Comment:
very nice way to get current date to evaluate it later using === trip.date. This works because they are both strings and you're strictly evaluating them.

In order to refactor your find yearly spent function using a dynamic date for today (using new Date()) you're going to need to figure out how to subtract a year from new Date(), create a new date object using that date, turn your trip.dates into Date objects and then evaluate them (filter all trips with a trip.date greater than a year ago; put another way: filter all trips with a trip.date Date object greater than a Date object a year ago). Whew!